### ISSUE TITLE
Regex exclude not working for actionlint

### ISSUE THREAD

**Issue by msha1026** [oxsecurity/megalinter#2295]
**Describe the bug**
When setting `ACTION_ACTIONLINT_FILTER_REGEX_EXCLUDE` in `.mega-linter.yml` to a specific file or even a directory, the file(s) are not excluded from actionlint linting. I have tried the same regular expression with other enabled linters like `JSON_JSONLINT_FILTER_REGEX_EXCLUDE` and it works as expected. Specifically, we set it to `(.github/workflows/cd.yml)`.

**To Reproduce**

1. Add `ACTION_ACTIONLINT_FILTER_REGEX_EXCLUDE: <some regex for your filesystem>` to `.mega-linter.yml`
2. Force a failure in one of the files that should be excluded
3. Run megalinter and see that the failure still shows

**Expected behavior**
Expect that the file(s) specified by `ACTION_ACTIONLINT_FILTER_REGEX_EXCLUDE` should be excluded when running actionlint.

**Comment by nvuillam** [comment]
action is with cli_lint_mode list_of_files, so filtering regex should work

Please could you share your configuration variable (with the regex)  and the file name ?

**Comment by msha1026** [comment]
Right now, we tested the configuration variable with a basic regex - `ACTION_ACTIONLINT_FILTER_REGEX_EXCLUDE: (.github/)`. The file that should be skipped has the following path: `.github/workflows/cd-github-release.yml`.

**Comment by nvuillam** [comment]
With regex you have to escape the dots: try (\.github/) :)

**Comment by nvuillam** [comment]
\\.github :)

**Comment by msha1026** [comment]
That does not work either, unfortunately.

It's interesting because we have a very similar regex exclusion with jsonlint. Our configuration variable for exclusions in jsonlint is `JSON_JSONLINT_FILTER_REGEX_EXCLUDE: (.vscode/)`. This filtering works as expected. When that configuration variable is present in the mega linter configuration, jsonlint errors in `.vscode` directory are ignored. So, we followed the same exact pattern for `ACTION_ACTIONLINT_FILTER_REGEX_EXCLUDE` with the `.github` directory, and it does not work the same way.

I have also tried to see if other linters can successfully exclude our `.github` directory. For example, I force a jsonlint error in a json file in our `.github` directory. I execute `mega-linter-runner -e 'ENABLE_LINTERS=JSON_JSONLINT'` locally, and see the error as expected. Then I add `JSON_JSONLINT_FILTER_REGEX_EXCLUDE: (.github/)` to the megalinter configuration, run mega linter again, and do not see the failure anymore. This means that the `.github` directory was excluded successfully for jsonlint.

So, I'm not sure why this same exclusion works for other linters like jsonlint, but it is not working for actionlint. Is it because actionlint will naturally search the `.github` directory since that is where all github actions reside? So it may be ignoring my `.github/` exclusion? (I'm not totally sure how actionlint works, so that is just a guess). I'll be digging into this a bit more, but thought I'd bring it up in case others have come across this :)
